packages/apm/src/integrations/tracing.ts

```diff
@@ -352,6 +519,32 @@ export class Tracing implements Integration {
     }
   }
 
+  /**
+   * Adds offset to the span
+   *
+   * @param measureName name of the performance measure
+   * @param span Span to add data.offset to
+   */
+  private static _addOffsetToSpan(measureName: string, span: SpanClass): void {
+    if (global.performance) {
+      const name = `#sentry-${measureName}`;
```

Comment:
Why the `#` in front of `sentry`?

Reply:
to make it "private" :D

Reply:
This is a global namespace (within the perf entries), `#sentry-foo` is no more private than `sentry-foo`, is it?

Reply:
Nope. Likely nobody should use this prefix.

Reply:
I'd like to use a "special char" in order if we ever need to do any string matching. No other reason than my personal style to prefix it with `#` ^^